capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,90 @@
+package com.google.univiz;
```

Comment:
Per the standup discussion, here is what we agreed this data structure should do:

 - It should capture the scorecard return data in its pure form with no extra interpretation.
 - It should have all the fields that our webapp would need for the mvp features and no extra fields.
 - The field names should match the scorecard field name so gson can deserialize correctly.

Reply:
+1 - can you also add a unit test that verifies that json from the service can be parsed by your new autovalue class?

The sample json input can live in `src/test/resources/com/google/univiz/scorecard`.

Reply:
ive added a unit test in #48.

Reply:
Sounds good - FWIW going forward it will be nice to have tests in the same PR as the class under test. That way we can verify that the test WAI and can request new test cases while reviewing the code...

Reply:
(I'm also fine with you pulling the test into this PR)

Reply:
ohhhh okay. that makes sense. i will make sure to put tests in the same PR in the future instead of pulling separate ones. (ill merge them though!)